databricks/koalas/tests/test_indexes.py

```diff
@@ -140,6 +140,21 @@ def test_multi_index_names(self):
         with self.assertRaises(PandasNotImplementedError):
             kidx.name = 'renamed'
 
+    def test_index_unique(self):
```

Comment:
Sorry if I missed but did we have a test case for multi-index?

Reply:
no, i think their implementation is slightly different, and i thought the scope of this PR is for Index, not MultiIndex (I was supposed to do in a separate PR).

But do you prefer to have this `unique` function added for both Index and MultiIndex in this PR? @HyukjinKwon

Reply:
Ah, it's fine to target to implement it only in `Index`. Can we explicitly throw a notimplemented exception manually?

Reply:
i added a `PandasNotImplementedError`, pls let me know if this is what you meant, thanks.